.github/workflows/Issue_checklist_checker.yml

```diff
@@ -5,7 +5,7 @@ on: issues
 jobs:
   checkIssue:
     name: Check Issue for Incomplete Checklist
-    runs-on: ubuntu-latest
+    runs-on: self-hosted
```

Comment:
Why does this say self-hosted, when all of the others run on "vds2019x11"?

Reply:
Because VSC file search (& replace) sucks